Wiley Online Library.js

```diff
@@ -262,15 +262,7 @@ function scrapeBibTeX(doc, url) {
 		translator.setString(text);
 
 		translator.setHandler('itemDone', function(obj, item) {
-			// BibTeX throws the last names and first names together
-			// Therefore, we prefer creators names from EM (if available)
-			var authors = doc.querySelectorAll('meta[name="citation_author"]');
-			if (authors && authors.length>0) {
-				item.creators = [];
-				for (let i=0; i<authors.length; i++) {
-					item.creators.push(ZU.cleanAuthor(authors[i].content, 'author'));
-				}
-			}
+			
```

Comment:
How can we be sure that this is not needed in a different context?

Reply:
i have taken the change from the master file.

Reply:
see also changed test results.

Reply:
see above.